src/frontend/src/components/public/PIAAdditionalRisks/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+import Messages from './messages';
+import InputText from '../../common/InputText/InputText';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus, faTrash } from '@fortawesome/free-solid-svg-icons';
+import { useNavigate } from 'react-router-dom';
+import List from '../../common/List';
+
+export interface TableColProps {
```

Comment:
This interface is identical to the above InputTextProps and I believe that it is used for the same data. I would recommend using this interface in the List **OR** using the List's InputTextProps interface.